app.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express")
+const app = express()
+app.use(express.json())
+const {getTopics} = require('./controllers/topics.controllers')
+
+app.get('/api/topics', getTopics)
+
+app.use((err, req, res, next)=>{
+    if (err.status) {
+        console.log(err.status)
+        res.status(err.status).send({ msg: err.msg })
+    }
+    else next(err)
+})
```

Comment:
It's great you're thinking of error handling, but please only add the logic when you are testing for the error. Otherwise, your tests will pass straight away!